sfge/src/main/java/com/salesforce/rules/PathBasedRuleRunner.java

```diff
@@ -86,7 +86,28 @@ public Set<Violation> runRules() {
         return violations;
     }
 
-    private void executeRulesOnAnomalies(List<PathExpansionException> anomalies) {}
+    private void executeRulesOnAnomalies(List<PathExpansionException> anomalies) {
+        // If no anomalies were reported, we should log that information and just exit.
+        if (anomalies.isEmpty() && LOGGER.isInfoEnabled()) {
```

Comment:
If log level is at warn wouldn't this continue? Can we break the conditions into separate IF-clauses?

Reply:
I suppose the for-loop would be a no-op unlike a Stream foreach. We might be okay.
But seems like a good practice to keep Logger.enabled check separate in general.

Reply:
Good catch. I'll break it into separate clauses.